src/tools/sendEmail/__tests__/sendEmail.test.ts

```diff
@@ -22,13 +22,13 @@ describe("sendEmail", () => {
 
   beforeEach(() => {
     jest.clearAllMocks();
-    (client.send as jest.Mock).mockResolvedValue(mockResponse);
+    (client!.send as jest.Mock).mockResolvedValue(mockResponse);
```

Comment:
instead of banging every reference to the client, I'd rather bang it once in `client.ts` and ensure the `client` is present... tests would fail otherwise, anyway

Reply:
or if this is to run only a subset of tests, then declare its type in `client.ts` as present, and do a `ts-ignore`  there